Dockerfile

```diff
@@ -97,13 +97,11 @@ COPY conan/settings.yml "${HOME}/.conan/settings.yml"
 COPY conan/registry.txt "${HOME}/.conan/registry.template.txt"
 
 RUN groupadd --gid 1000 captain \
-  && useradd --home-dir "$HOME" \
-    --uid 1000 --gid 1000 \
-    captain \
-  && mkdir --parents \
-    $HOME/.ssh \
-  && chown --recursive captain:captain "$HOME" \
-  && chmod --recursive 777 "$HOME" \
+  && useradd --home-dir "${HOME}" --uid 1000 --gid 1000 captain \
+  && mkdir --parents "${HOME}/.ssh" \
+  && cp /root/.bashrc /root/.profile "${HOME}" \
```

Comment:
This is the only line I really changed. The rest of it is just deleting whitespace that I didn't find helpful at all.